pages/common/crontab.md

```diff
@@ -2,6 +2,7 @@
 
 > Schedule cron jobs to run on a time interval for the current user.
 > Job definition format: "(min) (hour) (day_of_month) (month) (day_of_week) command_to_execute".
+> To avoid mistake, use [crontab guru](https://crontab.guru).
```

Comment:
While this is helpful, I am not sure content like this falls within the scope of tldr pages. This is more appropriate in other developer content related websites IMO.

Reply:
I would suggest placing it in tldr pages as it just makes everything so much easier.

Reply:
I think it's not our job to promote any specific tool, especially not on other tool's page. If crontab guru has a CLI tool, it can be made into its own page.